third-party/qthread/qthread-1.10/src/interfaces/chapel/tasks-qthreads.h

```diff
@@ -411,5 +411,16 @@ c_sublocid_t chpl_task_getRequestedSubloc(void)
     return c_sublocid_any;
 }
 
+#ifdef CHPL_TASK_TASKS_BOUND_TO_PTHREADS_IMPL_DECL
+#error "CHPL_TASK_TASKS_BOUND_TO_PTHREADS_IMPL_DECL is already defined!"
+#else
+#define CHPL_TASK_TASKS_BOUND_TO_PTHREADS_IMPL_DECL 1
+#endif
+extern int chpl_qthread_tasks_bound_to_pthreads;
+static inline
+int chpl_task_tasksBoundToPthreads(void) {
+  return chpl_qthread_tasks_bound_to_pthreads;
```

Comment:
For the first question: My understanding is that the THREADQUEUE_POLICY stuff just talks about number of workers per shepherd (queue), and not about migration as such.  In particular, it's conceivable in some scheduler that a task/qthread could migrate from one shepherd to another. So, I'd like to continue depending on the scheduler for this decision.

For the second: Yes, that would actually be safer. I do have a table (in one of my committed meeting notes) based on a diagram Dylan put up on a whiteboard describing the schedulers and whether or not each one can steal, but basing this decision on the one we definitely know is safe instead of the one we definitely know is unsafe would still be better.